src/diy/README.md

```diff
@@ -1,9 +1,9 @@
 # DIY Trackers Guide
-So you want to build your own SlimeVR trackers?
+This guide is to help you build your trackers from scratch. Please not if you have purchased the DIY Kit from SlimeVR, please visit the [DIY Kit page](../diy_kit_guide.html).
```

Comment:
```suggestion
This guide is to help you build your trackers from scratch. Please note: if you have purchased the DIY Kit from SlimeVR, please visit the [DIY Kit page](/diy_kit_guide.html).
```

Reply:
I actually had a big think about relative for absolute directory linking here and ended up going for relative as before mdBooks there was a duplicate version of the documentation being spawned at a 'default' url, which didn't have it in root directory. I didn't want someone who found that  to be like 'oh the docs don't work'. (The not -> note is of course something I'll change)